custom/icds_reports/migrations/0113_merge_20190424_1803.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# flake8: noqa
+# Generated by Django 1.11.20 on 2019-04-24 18:03
+from __future__ import absolute_import, unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
This seems unnecessary - can't you just change the migration in this PR so that it doesn't conflict?

Reply:
I agree. I've been prioritizing merges over replacing migrations since there's been so many migrations recently. Will replace for final merging

Reply:
fixed https://github.com/dimagi/commcare-hq/pull/24098/commits/7deeff81fb0d85350b0d602d8fccda43a627e84b and again https://github.com/dimagi/commcare-hq/pull/24098/commits/377622e8c898c3d4aff89988e8dc5c91a7b8b03c